testing/injections/timers_disabled_shim.js

```diff
@@ -0,0 +1,13 @@
+// ts file compiled js file somehow end up with cjs format but esbuild needs esm format for shims.
```

Comment:
This files are both in the `testing` directory--just so I understand, these shims happen at compile time, and not at runtime in creation of the isolate? So if I used setTimeout() from the web editor, I would just get an error that it's not defined?

(Totally separate, we should move the compiler stuff into its own directory instead of `testing` at some point.)

Reply:
yes, it's only at the compile time. 

re web editor: the current web editor compile flow is missing a few things from the CLI, e.g. shims or other browserify stuff. I've already seen issues with that and have a ticket to track. 

re: testing/compile. totally. let me do that later. I am currently in a big mess of changes to debug the drive pack.

Reply:
Yep re moving the files, just flagging, definitely not urgent.